.github/workflows/scan-licenses.yml

```
@@ -0,0 +1,21 @@
+name: Scan licenses
+on:
```

Let's keep it tidy:
```suggestion
name: Scan licenses

on:
```

and let's also add `concurrency` setup: https://github.com/infinum/default_rails_template/blob/ce3d22884a1e32c9fcf0ad93ddc8db1ed5939ca6/.github/workflows/publish-docs.yml#L3-L5